cluster/v1beta1/types_placement.go

```diff
@@ -94,6 +95,41 @@ type PlacementSpec struct {
 	// certain taints to be selected by placements with matching tolerations.
 	// +optional
 	Tolerations []Toleration `json:"tolerations,omitempty"`
+
+	// DecisionStrategy divide the created placement decision to groups and define number of clusters per decision group. Decision result will be sorted by name and filled into decisions if it is not set.
+	// +optional
+	DecisionStrategy DecisionStrategy `json:"decisionStrategy,omitempty"`
+}
+
+// DecisionGroup define a subset of clusters that will be added to placementDecisions with groupName label.
+type DecisionGroup struct {
+	// Group name to be added to the created placement Decisions labels with label key cluster.open-cluster-management.io/decision-group-name
```

Comment:
Group name to be added as the value of the created PlacementDecisions labels with key ...